spec/takeaway_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'takeaway'
+
+describe TakeAway do
+
+  subject(:takeaway) {described_class.new menu_klass, sms_klass}
+  let(:menu) {double('menu', show_menu: [:pizza, 2], add: nil, dishes: {pizza: 10} )}
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)